test/e2e/docs/overview.md

```diff
@@ -29,13 +29,18 @@ To accelerate development by being a force for continuous improvement, and help
 
 ## What is tested?
 
-At the high level, each test file (or `spec`) fall under one of the following flows:
-
-| Flow                 | Directory                |
-| -------------------- | ------------------------ |
-| Calypso              | `specs/specs-playwright` |
-| Editor               | `specs/specs-wpcom`      |
-| Internationalization | `specs/specs-i18n`       |
-| Jetpack              | `specs/specs-jetpack`    |
-
-Core code for Jetpack, WooCommerce and Gutenberg are hosted in other repositories and they have separate e2e testing infrastructure. Tests within `test/e2e` are meant to test interactions between their respective components and Calypso/WordPress.com.
+Each test file (referred to as `spec`) should be assigned to at least one group.
```

Comment:
What do you think about a link here to the instructions for [how to assign something to a test group](https://github.com/Automattic/wp-calypso/blob/update/playwright-documentation/test/e2e/docs/writing_tests.md#quick-start )? People might be curious here how that happens. Or we could at least mention that files are assigned through a jsdoc header!

Reply:
I've moved the test group discussion to `Running tests in CI`. 

Doesn't the Writing Tests > Quick Start step cover the process of assigning the test group?